### DISCUSSION TITLE
修改书签的对齐方式？

### DISCUSSION THREAD

**Discussion by retsyo** [wmjordan/PDFPatcher#230] [问答]
# 1. 情况描述
今天我碰到了一个奇怪的简体中文pdf，在sumatrapdf中打开时书签是右对齐的，在foxit/wps/补丁丁中是左对齐的

# 2. 问题/需求
补丁丁可以修改书签的对齐方式吗？

补：在以上pdf阅读器中，sumatrapdf可以F2然后改名

thanks

**Comment by wmjordan** [comment]
不考虑右对齐书签。

**Reply by retsyo** [reply]
那可以将右对齐的修改回来吗？

**Reply by wmjordan** [reply]
试试保存PDF文档，在“阅读方式”中将“阅读方向”选为“从左到右”。
如果不行，试试用提取页面功能，在页码处输入该文档的页码范围，重新生成一个PDF文档。

**Reply by retsyo** [reply]
好的，下次我再碰到这样的PDF时，我试试
